ui/appui-react/src/appui-react/layout/footer/message-center/Message.scss

```diff
@@ -3,19 +3,15 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 @import "~@itwin/core-react/lib/cjs/core-react/style/themecolors";
```

Comment:
If you renamed the `Message.tsx` file to `MessageCenterMessage.tsx` then you should probably rename this one too for consistency and clarity.